app/controllers/wcc/contentful/application_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Mime::Type.register 'application/vnd.contentful.management.v1+json', :json
```

Comment:
I don't think this belongs here... can we put in an initializer or engine specific config file(s)?

Reply:
http://edgeguides.rubyonrails.org/configuring.html#initializers